source/administration/changelog.md

```diff
@@ -4,6 +4,133 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.14 - [Feature Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
+
+Release Date 2019-08-16
+
+### Compatibility
+
+#### Removed and Deprecated Features
+
+ - We are removing support for Internet Explorer (IE11) in Mattermost v5.16.0, which releases on October 16, 2019. Learn more in our [forum post](https://forum.mattermost.org/t/mattermost-is-dropping-support-for-internet-explorer-ie11-in-v5-16/7575).
+ - **Enable Plugin Health Check** removed from ``config.json``. Instead, all plugins are now automatically monitored and restarted if a plugin has crashed or is deemed unhealthy. If a plugin crashes three times within an hour, the server automatically deactivates the plugin.
+ 
+### Breaking Changes
+ - **Change related to viewing webhooks**
+
+**IMPORTANT:** If you upgrade from a release earlier than 5.12, please read the other [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html).
+
+### Highlights
+
+#### Guest Accounts (EE)
+#### Synchronize LDAP Groups to Teams/Channels (EE)
+#### ADA screen reader accessibility improvements
+#### Added bidirectional scrolling to land on oldest unread post
+#### Jira V2.1
+#### Pre-packaged plugins (Jenkins, Walltime)
```

Comment:
@aaronrothschild @crspeller can you please provide the exact list of new bundled plugins for 5.14?

Reply:
Walltime was cut. Jenkins, antivirus, and Skype for business are in progress but not in yet.